lib/Dialect/FIRRTL/Transforms/Dedup.cpp

```diff
@@ -250,6 +250,14 @@ struct Equivalence {
     if (aType.isa<BundleType>() && bType.isa<BundleType>())
       return check(diag, message, a, aType.cast<BundleType>(), b,
                    bType.cast<BundleType>());
+    if (aType.isa<RefType>() && bType.isa<RefType>())
+      return (check(diag,
+                    message + ", mismatch in RefType (can be due to difference "
```

Comment:
Mega nit: can you make these more complete sentences as opposed to fragments?

Consider:

"has a RefType port with a different base type ($type) of a port in the same position on a different module and these modules were marked as "must dedup". (This may be due to Grand Central Taps or Views being different between the two modules.)"

The note could then be:

"module port '$name' has different base type ($type)"